### PR TITLE
Fix subservices

### PR DESCRIPTION
**Pull request by nfamula**
- switched to prismic
- added subservices as a new field
- moved content in prismic to subservices field

### CODE REVIEW

File: src/components/Heading.js [hunk 1]
```diff
@@ -1,64 +0,0 @@
-import React from 'react'
-
-const HEADINGS = {
-  h1: {
```
Comment (Potato-coder) [inline src/components/Heading.js]: Just curious what was this file used in before

Comment (nfamula) [reply src/components/Heading.js]: I think this was added earlier in today by @JakeMaldonado

Comment (Potato-coder) [reply src/components/Heading.js]: Thanks!  A bunch of helmet stuff also got removed probably accidental too? e.g index.js

Comment (nfamula) [reply src/components/Heading.js]: @Potato-coder I just copied over all of what was in your /src on the prismic branch

Comment (Potato-coder) [reply src/components/Heading.js]: Oh Sorry I meant the react helmet stuff was it sth @JakeMaldonado added and it got overwritten should have clarified my bad. I will leave it to him to decide on those stuff if it’s important or not. From prismic side its good

Comment (nfamula) [reply src/components/Heading.js]: ohh okay! thanks @Potato-coder I can add back anything if its been removed @JakeMaldonado lmk :)

Comment (nfamula) [reply src/components/Heading.js]: I've added back the react-helmut and the headings in services.js 
can we merge this in? @JakeMaldonado

File: src/components/services.js [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
-  }
-  `
-  )
+    }  
+  `)
 
   return (
     <div>
-      <Heading level={1} text="Home Improvement Services" />
+      <h1>Home Improvement Services</h1>
```
Comment (JakeMaldonado) [inline src/components/services.js]: I think we should leave this and the `h1` below as a heading component if we can

Comment (nfamula) [reply src/components/services.js]: missed that one, will fix!

Comment (JakeMaldonado) [reply src/components/services.js]: Awesome thanks, looks good

Comment (nfamula) [reply src/components/services.js]: thanks! :)

**Pull request merged**